### ISSUE TITLE
Undefined Variable

### ISSUE THREAD

**Issue by lovetox** [Legrandin/pycryptodome#20]
number.py

```
# You need libgmp v5 or later to get mpz_powm_sec.  Warn if it's not available.
if _fastmath is not None and not (hasattr(_fastmath, "HAVE_DECL_MPZ_POWM_SEC") and _fastmath.HAVE_DECL_MPZ_POWM_SEC):
    _warn("Not using mpz_powm_sec.  You should rebuild using libgmp >= 5 to avoid timing attack vulnerability.", PowmInsecureWarning)
```

PowmInsecureWarning is Undefined

**Comment by battyc** [comment]
It looks like you need the latest libgmp installed.  That warning indicates all versions predating libgmp 5 are vulnerable to a timing attack so it won't allow you to run without a warning.  Try installing libgmp3-dev and then giving it another shot.

**Comment by lovetox** [comment]
no, this is not about libgmp, there is an undefinded variable in the code

"PowmInsecureWarning"
what is that, where is it defined?

**Comment by battyc** [comment]
My apologies, you were correct.  After reviewing the commit history for that section of code it looks like the mentioned Warning was removed.  That segment of code should definitely just be commented out or removed since _fastmath is manually set to "None" and therefore the if statement containing the warning you mentioned will never run.


See the commit where the warning was removed [here](https://github.com/Legrandin/pycryptodome/commit/ec7f3c8662b5195bf0afe25c8c42ce32e0737eea#diff-ec2c39f57bafbda9193f7bc0017692cf).

**Comment by Legrandin** [comment]
Fix is in master (df733cd3a3f531ec4e1304968db7d99d8e49ca05).
